docs/team/seanflyyy.md

```diff
@@ -9,18 +9,36 @@ Teacher’s Pet is a desktop application for managing contacts of students and c
 
 Given below are my contributions to the project.
 
-* **New Feature**: Added the ability to find the next available class
-  * What it does: outputs the next available class based on the specified time range and duration of class
-  * Justification: This feature is an integral part of this application as without it, it will be really tedious 
-  for the user to find an available class, and the schedule feature of the application will not be complete.
-  * Highlights: Finding the next available class in the specified requirements is much faster than manually checking 
-  each day until the user finds the slot he/she wants.
-
 * **New Feature**: Added find by tag feature
+  * What it does: allows searching up of students based on their tags
+  * Justification: This feature allows for users to find students based on tags which is an integral part of our
+  application. Without this feature, the tags would be much less helpful.
+  * Highlights: The user is able to input multiple tags in order to find students who contain more than one tag.
 
 * **New Feature**: Added find by next of kin phone feature
-* 
+  * What it does: This feature allows for users to find students based on their next of kin phone number, providing 
+  another way for users to find their students.
+  * Justification: This feature provides users with an alternative way of finding students, making the find command
+  more dynamic.
+  * Highlights: The find by next of kin phone feature would search for the exact match of the student's next of kin 
+  phone number.
+
 * **New Feature**: Add Schedule List Panel and styled UI
+  * What it does: Designed the UI to improve readability and user experience. Added Schedule List Panel in order to
```

Comment:
I feel that "Designed the UI..." is not describing what it does but rather what you did.